src/app/app.component.ts

```diff
@@ -69,6 +67,5 @@ export class AppComponent implements OnInit {
   ForgotPassword(userName) {
     this.authService.ForgotPassword(userName);
     this.modal.close();
```

Comment:
Lets make a convention for Function names to be in camel case, try to define them as `forgotPassword` and not `ForgotPassword`